packages/pds/src/db/util.ts

```diff
@@ -27,6 +28,14 @@ export const softDeleted = (repoOrRecord: { takedownId: number | null }) => {
 
 export const countAll = sql<number>`count(*)`
 
+export const nullToZero = (db: Database, ref: DbRef) => {
+  if (db.dialect === 'pg') {
+    return sql<number>`coalesce(${ref}, 0)`
+  } else {
+    return sql<number>`ifnull(${ref}, 0)`
+  }
```

Comment:
I think sqlite actually might also support `coalesce()`!

Reply:
ah sweet think you're right 👍